src/components/projecteditor/style.less

```diff
@@ -141,15 +133,18 @@
     }
 
     .sideButton {
-        border-color: transparent;
         padding: 5px;
         transition: all .15s ease;
-        border-radius: 0px;
```

Comment:
Is this one intended? Buttons in panels now have border-radius again after removing this style

Reply:
Actually not! Let me remove that one

Reply:
I will remove it from the btnNoBg global style instead.